app/scripts/configs/datatype-to-track-type.js

```diff
@@ -3,7 +3,26 @@ import { TRACKS_INFO } from '.';
 export const DATATYPE_TO_TRACK_TYPE = orientation => {
   const localDatatypeToTrackType = {};
 
-  TRACKS_INFO.filter(x => x.orientation === orientation).forEach(ti => {
+  // can a track be placed in a given orientation
+  const orientationMatches = x => {
+    // if the track's orientation matches the given orientation, then yes
+    if (x.orientation === orientation) {
+      return true;
+    }
+
+    if (
+      orientation === '1d-vertical' &&
+      x.orientation === '1d-horizontal' &&
```

Comment:
What about the other way around? `orientation` is `1d-horizontal`?

Reply:
The default  orientation for tracks is `1d-horizontal` so if the selected orientation is `1d-horizontal` and  the  track's orientation is  `1d-horizontal` then we're ok. Otherwise we can't place the track in the selected orientation. That's a bit confusing but I think it's correct.